README.md

```diff
@@ -1,623 +1,1519 @@
 
   
 
+  
+
 Service also uses gotenberg for creation of project certificate. You can read more about it here : [Gotenberg](https://gotenberg.dev/docs/getting-started/introduction).
 
   
 
+  
+
 ## Installation
 
   
 
-1.  **Create project Directory:** Create a directory named **project**.
+  
+
+1.  **Create survey Directory:** Create a directory named **survey**.
+
+  
 
   
 
 > Example Command: `mkdir survey && cd survey/`
 
   
 
-2.  **Download Docker Compose File:** Retrieve the **[docker-compose.yml](https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/docker-compose-project.yml)** file from the Project service repository and save it to the project directory.
+  
 
-```
+2.  **Download Docker Compose File:** Retrieve the **[docker-compose.yml](https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/docker-compose.yml)** file from the Project service repository and save it to the project directory.
 
-curl -OJL https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/docker-compose-project.yml
+  
 
 ```
-> Note: All commands are run from the project directory.
+curl -OJL https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/docker-compose.yml
+```
+
+> Note: All commands are run from the survey directory.
+
+  
 
   
 
 Directory structure:
 
+  
+
 ```
 
+  
+
 ./survey
 
+  
+
 └── docker-compose.yml
 
+  
+
 ```
+
 3.  **Download Environment Files**: Using the OS specific commands given below, download environment files for all the services.
 
   
 
+  
+
 -  **Ubuntu/Linux/Mac**
 
+  
+
 ```
 
+  
+
 curl -L \
 
+  
+
 -O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/interface_env \
 
+  
+
 -O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/entity_management_env \
 
+  
+
 -O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/samiksha_env \
 
+  
+
 -O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/notification_env \
 
+  
+
 -O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/scheduler_env \
 
+  
+
 -O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/user_env \
 
--O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/env.js
+  
 
+-O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/env.js
 ```
 
--  **Windows**
-
-```
+  
 
-curl -L ^
+-  **Windows**
```

Comment:
@borkarsaish65 it subheading correct alignment

Reply:
corrected